client/src/test/java/tds/config/TimeLimitConfigurationUnitTests.java

```diff
@@ -0,0 +1,30 @@
+package tds.config;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TimeLimitConfigurationUnitTests {
```

Comment:
It doesn't matter to me but I usually just call my unit tests TimeLimitConfigurationTest which to my knowledge is more of the norm in Java.  However, I don't fee strongly but we may want to agree on convention so projects look similar.